lib/view/icu_non_intensivist/ventilation/adjuncts_view.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:wh_covid19/view/view_templates/html_text_card_view_template.dart';
+
+import '../../../static_data_repo.dart';
```

Comment:
Use absolute import references if possible (package:wh_covid19/...)

Reply:
ok, this was added automatically by Android Studio... is it a setting?